source/classes.tex

```diff
@@ -173,12 +173,14 @@
 
 $M(\mathtt{X})$ is defined as follows:
 \begin{itemize}
-\item If \tcode{X} is a non-union class type, the set $M(\mathtt{X})$ is
-empty if \tcode{X} has no (possibly inherited (Clause~\ref{class.derived}))
-non-static data members;
-otherwise, it consists of the type of the first non-static data member
-of \tcode{X} (where said member may be an anonymous union), $\mathtt{X}_0$,
-and the elements of $M(\mathtt{X}_0)$.
+\item If \tcode{X} is a non-union class type with no (possibly
+inherited (Clause~\ref{class.derived})) non-static data members, the set
+$M(\mathtt{X})$ is empty.
+
+\item If \tcode{X} is a non-union class type whose first non-static data
+member has type $\mathtt{X}_0$ (where said member may be an anonymous union),
```

Comment:
I don't even think this is a question of consistency. Both ways work and either is fine. I just wanted to point out that you've done this differently elsewhere, but I think going any further is busywork with limited value.